src/Modalizer.ts

```diff
@@ -330,19 +330,25 @@ export class ModalizerAPI implements Types.ModalizerAPI {
     private _aug: WeakRef<HTMLElement>[];
     private _hiddenUpdateTimer: number | undefined;
     private _alwaysAccessibleSelector: string | undefined;
+    private _accessibleCheck: Types.ModalizerElementAccessibleCheck | undefined;
 
     activeId: string | undefined;
     currentIsOthersAccessible: boolean | undefined;
     activeElements: WeakRef<HTMLElement>[];
 
-    constructor(tabster: Types.TabsterCore, alwaysAccessibleSelector?: string) {
+    constructor(
+        tabster: Types.TabsterCore,
+        alwaysAccessibleSelector?: string,
```

Comment:
Let's add @deprecate on `alwaysAccessibleSelector`, there's no reason to have it with the check API

Reply:
I can just remove it and release next major version.

Reply:
I don't think it's a big enough change to warrant a major version, but I can leave that up to you

Reply:
So I'd prefer just to deprecate this, until we actually have a feature that's worth the cost of a major bump